daemon/lokinet-bootstrap.cpp

```diff
@@ -1,26 +1,17 @@
-#include <cpr/cpr.h>
 #include <llarp/constants/files.hpp>
 #include <llarp/constants/version.hpp>
-#include <llarp/util/buffer.hpp>
 #include <llarp/util/fs.hpp>
 
-#include <fstream>
-#include <sstream>
-#include <iostream>
+#include <cpr/cpr.h>
 
+#include <iostream>
 #include <unordered_map>
 #include <unordered_set>
 
 #ifndef _WIN32
 #include <openssl/x509.h>
 #endif
 
-#include <sstream>
```

Comment:
Dropping this seems suspicious, because we're using a stringstream in this file

Reply:
IWYU